File: ASCOM.Utilities/ASCOM.Utilities/AlpacaSupport/JSON Response Classes/AlpacaDescriptionResponse.vb
Public Class AlpacaDescriptionResponse
    Inherits RestResponseBase

    Public Sub New()
        Value = New AlpacaDeviceDescription()
    End Sub

    Public Sub New(ByVal clientTransactionID As UInteger, ByVal transactionID As UInteger, ByVal value As AlpacaDeviceDescription)
        ServerTransactionID = transactionID
        MyBase.ClientTransactionID = clientTransactionID
        Me.Value = value
    End Sub

    Public Property Value As AlpacaDeviceDescription
End Class

File: ASCOM.Utilities/ASCOM.Utilities/AlpacaSupport/JSON Response Classes/AlpacaDescriptionResponse.cs

namespace ASCOM.Utilities
{
    internal class AlpacaDescriptionResponse : RestResponseBase
    {

        public AlpacaDescriptionResponse()
        {
            Value = new AlpacaDeviceDescription();
        }

        public AlpacaDescriptionResponse(uint clientTransactionID, uint transactionID, AlpacaDeviceDescription value)
        {
            ServerTransactionID = transactionID;
            ClientTransactionID = clientTransactionID;
            Value = value;
        }

        public AlpacaDeviceDescription Value { get; set; }
    }
}